Formula/e/epinio.rb

```diff
@@ -33,9 +33,5 @@ def install
   test do
     output = shell_output("#{bin}/epinio version 2>&1")
     assert_match "Epinio Version: v#{version}", output
-
-    output = shell_output("#{bin}/epinio settings update-ca 2>&1")
-    assert_match "failed to get kube config", output
-    assert_match "no configuration has been provided", output
```

Comment:
The CLI is used to interact with an Epinio instance running on Kubernetes. Hard to test this.  

There are other similar tools that are doing the same because of this limitation:

## k9s

https://github.com/Homebrew/homebrew-core/blob/98bdff80b835112504723827f79774dd56cdb968/Formula/k/k9s.rb#L35-L38

## k3d

https://github.com/Homebrew/homebrew-core/blob/2a4368105c1257aefda1f7b4c3fa71c1d255ec15/Formula/k/k3d.rb#L46-L53

## k6

https://github.com/Homebrew/homebrew-core/blob/98bdff80b835112504723827f79774dd56cdb968/Formula/k/k6.rb#L28-L37

Reply:
> The CLI is used to interact with an Epinio instance running on Kubernetes. Hard to test this.

do we have any flag to inspect the environment??

Or is there config file that we leverage to just test out the CLI configurations part

Reply:
Hm, there is the `epinio settings show` command that is a local command to check the settings file that is going to be used. That should be enough.

I left the check on the `epinio version` though.

I'm not sure how much testing should be done about the functionalities of the binaries. But that's a different issue.